dapp/src/hooks/useLastUsedBtcAddress.ts

```diff
@@ -0,0 +1,28 @@
+import { useCallback } from "react"
+import useLocalStorage from "./useLocalStorage"
+
+export const LAST_USED_BTC_ADDRESS_KEY = "lastUsedBtcAddress"
```

Comment:
Can we keep the `LAST_USED_BTC_ADDRESS_KEY` unexported? We could define helper functions to read and write the last used btc address in the storage. Then we could use the read function in `dapp/src/wagmiConfig.ts`.

Reply:
Moved the discussion to https://github.com/thesis/acre/pull/708#discussion_r1784027460